src/Geometry/GMUGeometryRenderer.h

```diff
@@ -30,6 +30,10 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @interface GMUGeometryRenderer : NSObject
 
+/**
+ * The overlays array returned from the GMUGeometryRenderer, use after render function.
+ */
+- (NSArray<GMSOverlay *> *)mapOverlays;
```

Comment:
Please revert this change. The `mapOverlays` function defined in `GMUGeometryRenderer.m` was not designed to be exposed in the public interface. 

If you need access to this, I recommend creating a class extension similar to [GMUGeometryRenderer+Testing.h](https://github.com/googlemaps/google-maps-ios-utils/blob/master/src/Geometry/GMUGeometryRenderer+Testing.h) to expose this method.